c-dependencies/js-compute-runtime/xqd.wit

```diff
@@ -0,0 +1,510 @@
+
+      /**
+      * Country name.
+      *
+      * This field is the [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1) English short name for a country.
+      */
+      country-name: option<string>,
+
+      /**
+      * Time zone offset from Greenwich Mean Time (GMT) for `city`.
+      */
+      gmt-offset: option<string>,
+
+      /**
+      * Latitude, in units of degrees from the equator.
+      *
+      * Values range from -90.0 to +90.0 inclusive, and are based on the [WGS 84](https://en.wikipedia.org/wiki/World_Geodetic_System) coordinate reference system.
+      */
+      latitude: option<float32>,
+
+      /**
+      * Longitude, in units of degrees from the [IERS Reference Meridian](https://en.wikipedia.org/wiki/IERS_Reference_Meridian).
+      *
+      * Values range from -180.0 to +180.0 inclusive, and are based on the [WGS 84](https://en.wikipedia.org/wiki/World_Geodetic_System) coordinate reference system.
+      */
+      longitude: option<float32>,
+
+      /**
+      * Metro code, representing designated market areas (DMAs) in the United States.
+      */
+      metro-code: option<u32>,
+
+      /**
+      * The postal code associated with the IP address.
+      *
+      * These are available for some IP addresses in Australia, Canada, France, Germany, Italy, Spain, Switzerland, the United Kingdom, and the United States.
+      *
+      * For Canadian postal codes, this is the first 3 characters. For the United Kingdom, this is the first 2-4 characters (outward code). For countries with alphanumeric postal codes, this field is a lowercase transliteration.
+      */
+      postal-code: option<string>,
+
+      /**
+      * Client proxy description.
+      */
+      proxy-description: option<string>,
+
+      /**
+      * Client proxy type.
+      */
+      proxy-type: option<string>,
+
+      /**
+      * [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) country subdivision code.
+      *
+      * For countries with multiple levels of subdivision (for example, nations within the United Kingdom), this variable gives the more specific subdivision.
+      *
+      * This field can be None for countries that do not have ISO country subdivision codes. For example, None is given for IP addresses assigned to the Åland Islands (country code AX, illustrated below).
+      */
+      region: option<string>,
+
+      /**
+      * Time zone offset from coordinated universal time (UTC) for `city`.
+      */
+      utc-offset: option<u32>,
+    }
+
+    /// JSON string for now
+    geo-lookup: func(addr-octets: list<u8>) -> result<string, error>
+
+
+    /*
+     * Fastly KV
+     */
+    type kv-store-handle = u32
+
+    kv-open: func(name: string) -> result<kv-store-handle, error>
+
+    kv-lookup: func(store: kv-store-handle, key: list<u8>) -> result<option<body-handle>, error>
+
+    kv-insert: func(store: kv-store-handle, key: list<u8>, body-handle: body-handle, max-age: u32) -> result<bool, error>
```

Comment:
Same comment here as for backend-is-healthy, these definitions are not in the Viceroy wit files, which makes me think we shouldn't have them in js-compute-runtime yet